lib/pejs.ts

```diff
@@ -102,14 +103,37 @@ export class PEJS {
    * @param presentationDefinition the definition of what is expected in the presentation.
    * @param selectedCredential the credentials which were declared selectable by getSelectableCredentials and then chosen by the intelligent-user
    * (e.g. human).
+   * @param holderDID the decentralized identity of the wallet holder. This is used to identify the credentials issued to the holder of wallet.
    *
    * @return the presentation submission.
    */
-  public submissionFrom(
+  public presentationFrom(
     presentationDefinition: PresentationDefinition,
```

Comment:
The doc says it returns a presentation submission, but based on the function name and return type it's seems to be a presentation. Please update the doc.

Reply:
The docs are to be updated in wide range of projects. For that we used to have a separate ticket.
But in this project I will fix it.

Reply:
Fixed.